addon/manifest.json

```diff
@@ -7,8 +7,8 @@
   "description": "Track your web usage and generate a beautiful diagram. ",
 
   "icons": {
-    "48": "icons/icon.svg",
-    "96": "icons/icon.svg"
+    "48": "icons/icon-48.png",
+    "96": "icons/icon-96.png"
   },
 
   "browser_action": {
```

Comment:
almost, except that in ur `browser_action` section ur `default_icon` is still set to the old svg icon
```json
    "default_icon": "icons/icon.svg",
```
it should be set to ur new icon-48.png

Reply:
Just changed it. Is it automatically going to update the pull request?